components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -14,6 +15,38 @@ import "components/automate-grpc/protoc-gen-policy/api/annotations.proto";
 import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
 
 service Policies {
+  /*
+  Create a new policy
+
+  Creates a new IAM policy used to control permissions in Automate.
+  A policy is composed of one or more statements that act on a set of members.
+  Each statement contains a role as well as a list of projects.
+
+  The role defines a set of actions that the statement is scoped to.
+  The project list defines the set of resources that the statement is scoped to.
+  Pass `"projects": ["*"]` to scope a statement to every project.
+
+  The top-level projects list for the policy defines which project(s) the policy belongs to,
+  whereas the statement level projects list defines which project(s) the statement applies to.
+
+  This example creates a new policy in the (unassigned) project that grants the viewer role
+  on a few projects for all LDAP teams:
+
+  ```
+  {
+    "name": "My Viewer Policy",
+    "id": "viewer-policy",
+    "members": ["team:ldap:*"],
+    "statements" : [
+      {
+        "role": "viewer",
+        "projects": ["project1", "project2"]
+      }
+    ],
+    "projects": []
+  }
+  ```
```

Comment:
I'm not sure how to fix this, but the whitespace in this code fence is getting eaten.

![Screen Shot 2020-01-24 at 11 04 35 AM](https://user-images.githubusercontent.com/22962/73096317-54b09b00-3e99-11ea-9b85-0052848d11d5.png)

Reply:
i think that's true for _all_ the code examples we have in api docs